docs/dev/testing.rst

```diff
@@ -3,32 +3,24 @@
 ===============================================================================
 Testing PUDL
 ===============================================================================
-We use `Tox <https://tox.readthedocs.io>`__ to coordinate our software testing
-and to manage other build and sanity checking tools. Under the hood, it invokes
-a variety of other collections of command-line tools in predefined combinations
-that are described in ``tox.ini``. These include software tests defined using
-`pytest <https://pytest.org>`__, code linters like ``ruff``, documentation
-generators like Sphinx, and sanity checks defined as git pre-commit hooks. Each
-of these tools, or sometimes collections of related tools, can be selected at
-the command line. They can also be run independently without using Tox, but for
-the sake of simplicitly and standardization, we try to mostly just run them
-using the predefined settings we have configured in Tox.
-
-The simplest way to test PUDL -- which is also how the code is tested
-automatically by our continuous integration setup -- is to just run Tox alone
-with no arguments. This will typically take 25 minutes to run.
+We use `pytest <https://pytest.org>`__ to specify software unit & integration tests,
+and to coordinate data validation tests.  There are several ``pytest`` commands stored
+as targets in the PUDL ``Makefile`` for convenience and to ensure that we're all running
+the tests in similar ways by default.
+
+To run the software unit and integration tests that will be run in our automated CI on
+GitHub, you can use the following command:
 
 .. code-block:: console
 
-    $ tox
+    $ make pytest-unit pytest-integration
```

Comment:
I thought about doing something like this but wasn't 100% sure how to arrange it without duplicating code.  But I guess just listing the targets there is the way to do it.

It could also include `pre-commit run --all-files` and `docs-build` to do all the things that will get run in CI.